### PR TITLE
Enabling Auto-Auth by Default for Local Users

### PR DESCRIPTION
**Pull request by clintonb**
This functionality is safe for local usage. Additionally, I have set a sane default for the prefix.

@rlucioni @jimabramson

### CODE REVIEW

File: ecommerce/user/views.py [hunk 1]
```diff
@@ -19,11 +19,10 @@ def get(self, request):
         if not getattr(settings, 'ENABLE_AUTO_AUTH', None):
             raise Http404
 
-        if not getattr(settings, 'AUTO_AUTH_USERNAME_PREFIX', None):
-            raise ValueError('AUTO_AUTH_USERNAME_PREFIX must be set.')
+        username_prefix = getattr(settings, 'AUTO_AUTH_USERNAME_PREFIX', 'auto_auth_')
 
         # Create a new user with staff permissions
-        username = password = settings.AUTO_AUTH_USERNAME_PREFIX + uuid.uuid4().hex[0:20]
+        username = password = username_prefix + uuid.uuid4().hex[0:20]
```
Comment (rlucioni) [inline ecommerce/user/views.py:25]: One of the tests, `test_prefix_invalid`, sets `AUTO_AUTH_USERNAME_PREFIX` to `None`, causing this line to fail.

Comment (clintonb) [inline ecommerce/user/views.py:25]: Updated.

**Comment by jimabramson** [comment]
:+1:

**Comment by rlucioni** [comment]
:+1:

**Pull request merged**